roles/variables/defaults/main.yml

```diff
@@ -1323,12 +1324,11 @@ control_center_export_certs: "{{ssl_mutual_auth_enabled}}"
 control_center_keytab_path: /etc/security/keytabs/control_center.keytab
 
 ### Name of listener used by C3 to talk to Kafka
-control_center_kafka_listener_name: >-
-  {%- if rbac_enabled|bool and auth_mode=='mtls' -%}
-    internal_token
-  {%- else -%}
-    internal
-  {%- endif -%}
+control_center_kafka_listener_name: internal
```

Comment:
where does this get used?

Reply:
can you share the config file getting generated in that case? that will clarify the need